Formula/libprelude.rb

```diff
@@ -23,12 +23,27 @@ class Libprelude < Formula
 
   depends_on "libtool" => :build
   depends_on "pkg-config" => :build
+  depends_on "python@3.11" => [:build, :test]
```

Comment:
Python seems to be for (optional) bindings which are actually broken in existing bottle due to commented issues. Similar to other formulae, can just make build/test dependency.

Reply:
👍